pkg/authorization/registry/clusterpolicy/strategy.go

```diff
@@ -67,12 +67,12 @@ func (strategy) ValidateUpdate(ctx apirequest.Context, obj, old runtime.Object)
 }
 
 // GetAttrs returns labels and fields of a given object for filtering purposes
-func GetAttrs(obj runtime.Object) (labels.Set, fields.Set, error) {
+func GetAttrs(obj runtime.Object) (labels.Set, fields.Set, bool, error) {
```

Comment:
> pls document the bool.

My post-rebase list has us rethinking our lives after manually updating 50 or identical methods to match the kube ones (also done manually by clayton).  Yay initializers